scripts/template_tester.py

```diff
@@ -81,14 +81,24 @@ def retest(api, provider_key, template):
     trackerbot.mark_provider_template(api, provider_key, template, tested=False)
 
 
+def check_tested(api, template_name, provider_key):
+    print(str(trackerbot.check_if_tested(api, template_name, provider_key)).lower())
+
+
 if __name__ == '__main__':
     parser = trackerbot.cmdline_parser()
     subs = parser.add_subparsers(title='commands', dest='command')
 
     parse_get = subs.add_parser('get', help='get a template to test')
     parse_get.set_defaults(func=get)
-    parse_get.add_argument('--request_type', dest='request_type',
-                           help='get a teamplate based on provider type')
+    parse_get.add_argument(
+        '--request_type', dest='request_type', help='get a teamplate based on provider type')
+
+    parse_check_tested = subs.add_parser(
+        'check_tested', help='check if template is tested at least once on given provider type')
+    parse_check_tested.set_defaults(func=check_tested)
+    parse_check_tested.add_argument('--request_type', dest='request_type', help='provider type')
```

Comment:
So this is a arg for the sub-command that matches the key and destination of the command 'get', request_type.

Below where check_tested is mapped, `args.request_type` is used - its not clear if that is --request_type from 'get' command arg list, or from the 'check_tested' sub-command arg list.

Implementation detail aside of which is actually passed, if two different values are supplied, I think its ambiguous in general, and check_tested should just use the existing request_type arg.

Thoughts?

Reply:
Hm, I am not quite following :) Some of the other sub-commands re-use the same arg name and args variable, like 'provider_key' arg used by 'parse_mark' and 'parse_latest' parsers. I was following the same convention here. We only call 1 subcommand at once so args.request_type wouldn't come from 'get' when the "check_tested" subcommand was used. What does your idea for using the existing arg look like?

Reply:
Ah! This is a result of me code reviewing while tired - I thought parse_check_tested was being added to parse_get as a sub of 'get', not as a separate command. duh.